x/emissions/types/params.go

```diff
@@ -106,10 +107,14 @@ func validateMinBondFactor(i interface{}) error {
 }
 
 func validateAvgBlockTime(i interface{}) error {
-	_, ok := i.(string)
+	v, ok := i.(string)
 	if !ok {
 		return fmt.Errorf("invalid parameter type: %T", i)
 	}
+	_, err := strconv.ParseFloat(v, 64)
```

Comment:
do we have a range for AveBlockTime, like AveBlockTime must at least > 0 ?

Reply:
96a24f1d9000439e9d33b0c7d7391be62dae6e7e